app/channels.go

```diff
@@ -213,6 +213,11 @@ func NewChannels(services map[product.ServiceKey]any) (*Channels, error) {
 	pluginsRoute.HandleFunc("/public/{public_file:.*}", ch.ServePluginPublicRequest)
 	pluginsRoute.HandleFunc("/{anything:.*}", ch.ServePluginRequest)
 
+	services[product.ChannelKey] = &channelsWrapper{
```

Comment:
This is overwriting the services[ChannelKey] instantiated in app/server.go:244   Not a big deal since it's overwriting with an equivalent instance.

Reply:
Any reason for this?

Reply:
Oh this one escaped me, I think we should remove this from being created in the `app/server.go` as it's a channels-related service.